README.md

```diff
@@ -1,262 +1,129 @@
-![Wallaroo Logo](wallaroo-logo.png)
+<p align="center"><a href="https://www.wallaroolabs.com/"><img src="wallaroo-logo.png" alt="WallarooLabs logo" width="400"/></a></p>
+<h2 align="center">Build and scale real-time applications as easily as writing a script</h2>
+
 ---
+[![CircleCI](https://circleci.com/gh/WallarooLabs/wallaroo.svg?style=shield)](https://circleci.com/gh/WallarooLabs/wallaroo)
+[![GitHub license](https://img.shields.io/badge/license-apache%202-blue.svg)][wallaroo-community-license-readme]
+[![GitHub version](https://badge.fury.io/gh/WallarooLabs%2Fwallaroo.svg)](http://badge.fury.io/gh/WallarooLabs%2Fwallaroo)
+[![IRC][irc-badge]][irc-link]
+[![Groups.io][group-badge]][group-link]
 
-Wallaroo is a fast, elastic data processing engine that rapidly takes you from prototype to production by eliminating infrastructure complexity.
+A fast, stream-processing framework. Wallaroo makes it easy to react to data in real-time. By eliminating infrastructure complexity, going from prototype to production has never been simpler.
 
-- [What is Wallaroo?][what is wallaroo section]
-- [Status][status section]
-- [Getting Started][getting started section]
-- [Documentation][documentation section]
-- [Getting Help][getting help section]
-- [How to Contribute][contribute section]
-- [License][license section]
-- [Frequently Asked Questions][FAQ]
-- [Additional Links][additional links]
-- [About this Repository][about this repository section]
+Answer our [30-second survey][survey-link], and be entered to win a Wallaroo T-Shirt.
 
 ## What is Wallaroo?
 
-
-Wallaroo is a fast and elastic data processing engine that rapidly takes you from prototype to production.
-
 When we set out to build Wallaroo, we had several high-level goals in mind:
 
 - Create a dependable and resilient distributed computing framework
 - Take care of the complexities of distributed computing "plumbing," allowing developers to focus on their business logic
 - Provide high-performance & low-latency data processing
-- Be portable (i.e., run on-prem or any cloud)
+- Language agnostic
```

Comment:
What's the intention of adding this point? I'm not sure it really adds to the messaging here.

Reply:
I removed this line. Like we talked about, it highlights a possible future and not what we currently have.